app/src/actions/workspace.js

```diff
@@ -215,7 +215,7 @@ function processLegacyWorkspace(data, dispatch) {
     endOuterDate = new Date(serializedStartDate[1]);
     startOuterDate = new Date(endOuterDate.getTime() - workspace.state.timeExtent);
     startInnerDate = startOuterDate;
-    endInnerDate = new Date(startInnerDate.getTime() + Math.min(workspace.state.timeExtent, 15778476000));
+    endInnerDate = new Date(startInnerDate.getTime() + Math.min(workspace.state.timeExtent / 2, 15778476000));
```

Comment:
Well if I understood correctly, when loading legacy workspaces, you'll always end up with inner range === outer range (since timeExtent is used for both). 
This allows the UI to be loaded with a state that makes it more obvious that you can move the inner brush (since it doesn't take up the whole width). 
But yeah, I agree that half of it is kind of arbitrary, let me know what you think works best.